### ISSUE TITLE
Julia syntax highlighting not working in notebooks created with Jupyterlab

### ISSUE THREAD

**Issue by odaom** [jupyterlab/jupyterlab#6826]
**Describe the bug**
When I create a new Julia notebook in JupyterLab, there is no syntax highlighting in the notebook. 

However, if I create a notebook in the regular Jupyter Notebook environment, and then open the notebook in JupyterLab, the syntax highlighting is working correctly. Also, if I duplicate a notebook in JupyterLab which has working syntax highlighting, the duplicate also has working syntax highlighting.

I started using Julia just a couple of days ago. Python highlighting works just fine in all cases.

**To Reproduce**

1. Start JupyterLab
2. Create new notebook with Julia kernel
3. Write code in cell
4. No syntax highlighting in cell

**Screenshots**
<img width="618" alt="No syntax highlighting in notebook" src="https://user-images.githubusercontent.com/42997653/61183514-88854b00-a642-11e9-94b2-10cb94dbf3f5.png">

<img width="616" alt="Syntax highlighting works for notebook created outside JupyterLab" src="https://user-images.githubusercontent.com/42997653/61183513-8622f100-a642-11e9-85cd-1c9a429db925.png">

Here are the JSON views of two notebooks – the first created in JupyterLab and without syntax highlighting, the second created in Jupyter Notebook and with working syntax highlighting. **Notice the difference at `language_info`.**

<img width="306" alt="Syntax highlighting not working" src="https://user-images.githubusercontent.com/42997653/61183803-ec5d4300-a645-11e9-90a8-169aef51b826.png">

<img width="335" alt="Syntax highlighting working" src="https://user-images.githubusercontent.com/42997653/61183801-ea937f80-a645-11e9-8300-fd5c946ac10d.png">

**Desktop**
- OS: MacOS Mojave 10.14.5
- Browser: have tried in safari 12.1, chrome 75.0, firefox 67.0
- JupyterLab 1.0.1
- Julia 1.1.1
- IJulia 1.18.1

**Comment by krassowski** [comment]
Sounds like #6760.

**Comment by odaom** [comment]
Manually editing the JSON section `language_info` to 

```JSON
"language_info": {
   "file_extension": ".jl",
   "mimetype": "application/julia",
   "name": "julia",
   "version": "1.1.1"
  }
```
circumvents the issue on a per-notebook basis.

**Comment by ian-r-rose** [comment]
Thanks for the nice writeup @odaom, and thanks for opening the relevant IJulia issue @krassowski 

@jasongrout do you think we should relax the requirement that info replies have a `status` field until this is fixed in IJulia?

**Comment by jasongrout** [comment]
Yes. I would say for now that an undefined status is assumed to be ok.

I think it makes sense to keep this relaxation until we have a very clear kernel message compliance test suite we can point people to.

**Comment by jasongrout** [comment]
Or maybe just have this relaxation on kernel info messages.

**Comment by jasongrout** [comment]
Fixed by #6860
